addons/godot-firebase/database/reference.gd

```diff
@@ -66,14 +66,14 @@ func set_pusher(pusher_ref: HTTPRequest) -> void:
     if not _pusher:
         _pusher = pusher_ref
         add_child(_pusher)
-        _pusher.connect("request_completed", self, "on_push_request_complete")
+        _pusher.request_completed.connect(Callable(self, "on_push_request_complete"))
 
 
 func set_listener(listener_ref: Node) -> void:
     if not _listener:
         _listener = listener_ref
         add_child(_listener)
-        _listener.connect("new_sse_event", self, "on_new_sse_event")
+        _listener.new_sse_event.connect(Callable(self, "on_new_sse_event"))
```

Comment:
I would rather see this as:

`_listener.new_sse_event.connect(on_new_sse_event)`